matrix/system_of_linear_equation_in_2_variables.py

```diff
@@ -0,0 +1,102 @@
+"""
+Theory:-
+    https://www.mathsisfun.com/algebra/systems-linear-equations-matrices.html
+    Cramer's rule for 2x2 matrix:-
+    https://www.chilimath.com/lessons/advanced-algebra/cramers-rule-with-two-variables
+    a1x + b1y = = d1
+    a2x + b2y = = d2
+"""
+
+
+def calculate_x_and_y(eq1: list[int], eq2: list[int]) -> None:
```

Comment:
Let's change the filename to `cramers_rule.py` so that it is easier to find and avoid future duplicates.
```suggestion
def calculate_x_and_y(equation_1: list[int], equation_2: list[int]) -> None:
```